bentoml/server/bento_sagemaker_server.py

```diff
@@ -0,0 +1,67 @@
+# BentoML - Machine Learning Toolkit for packaging and deploying models
+# Copyright (C) 2019 Atalaya Tech, Inc.
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from flask import Flask, Response, request
+
+
+def bento_sagemaker_api_wrapper(api):
+    def wrapper():
+        response = api.handle_request(request)
+        return response
+    return wrapper
+
+
+def setup_bento_service_api_route(app, api):
+    route_function = bento_sagemaker_api_wrapper(api)
+    app.add_url_rule(rule='/invocations/{}'.format(api.name), endpoint=api.name,
```

Comment:
I thought it has to be '/invocation' with sagemaker but if this `/invocation/API_NAME` subpath works it would be great. If it does not work, I think we can at least do two things here:

1. make API_NAME an option in the CLI, e.g. `bentoml serve-sagemaker ./SAVED_ARCHIVE_PATH --api-name='predict'`

2. when the option is not presented: if there is only one API returned from `get_service_apis` then just use it. Else if there are multiple APIs returned, throw an exception and tell the user about the '--api-name' argument and limitation with sagemaker

Reply:
done